src/connect-zbt-1/about-the-device/about-firmware-updates.md

```diff
@@ -8,7 +8,7 @@ zendesk:
 
 The Home Assistant Connect&nbsp;ZBT-1 ships with a full-fledged Zigbee firmware. *At this point, there are no known issues with the factory-flashed firmware.*
 
-Firmware updates for the Connect&nbsp;ZBT-1 are natively supported in Home Assistant 2025.4 and newer if you are running the ZHA integration or OTBR:
+Firmware updates for the Connect&nbsp;ZBT-1 are natively supported in Home Assistant 2025.4 and newer if you are running the <abbr title="Zigbee Home Automation">ZHA</abbr> integration or <abbr title="OpenThread Border Router">OTBR</abbr>:
```

Comment:
Could we implement a `{% abbr "ZHA" %}` and a yaml file with these instead?
That will simplify the markdown (by removing need of html in it), and make the resulting text consistent.

Reply:
thanks!